packages/ifpack2/src/Ifpack2_Details_ChebyshevKernel_def.hpp

```diff
@@ -332,7 +276,10 @@ chebyshev_kernel_vector
                                      use_beta,
                                      true>;
       functor_type func (alpha, w, d, b, A, x_colMap, x_domMap, beta, rows_per_team);
-      Kokkos::parallel_for (kernel_label, policy, func);
+      if(A.nnz()>10000000)
```

Comment:
How was this number chosen -- does it just follow what KokkosKernels does?

Reply:
Yup. That's the same number as here:
https://github.com/trilinos/Trilinos/blob/442a24ba155050971e7cba30fa2c22fdda9ebce9/packages/kokkos-kernels/src/sparse/impl/KokkosSparse_spmv_impl.hpp#L314-L331

Reply:
There was the work I did a while back, that showed the SpMV struggled based on NNZ **per row** (on Cuda);  Less so than total nnz. Christian Trott tuned that via the vector length.

I wonder if these parameters need to be set based on the average nnz pe row or simply the number of rows.  Raw nnz seems odd (don't all of the kernels operate on rows?).